src/ir_Daikin.h

```diff
@@ -1,18 +1,15 @@
 /* Copyright 2016 sillyfrog */
-#ifndef IRDAIKINESP_H_
-#define IRDAIKINESP_H_
+#ifndef IR_DAIKIN_H_
```

Comment:
Yes. It's a style guide thing. The cpplint'er picked it up.
https://google.github.io/styleguide/cppguide.html#The__define_Guard

As for why a trailing '_', no idea. It's beyond my limited c++ knowledge as to guess why.